### PR TITLE
Add cadl-ranch structure default sln

### PR DESCRIPTION
**Pull request by nisha-bhatia**
Fixes: https://github.com/microsoft/typespec/issues/3967

### CODE REVIEW

**Review by jorgerangel-msft — COMMENTED** [review]
hey @nisha-bhatia - looks like this pr is missing the actual test file to test the generated test project and also the update to the `launchSettings.json` file to ensure the cadl ranch test can be executed through the debugger. Here are some of the existing tests for reference: https://github.com/microsoft/typespec/tree/main/packages/http-client-csharp/generator/TestProjects/CadlRanch.Tests/Http/Client/Structure and the [doc](https://github.com/microsoft/typespec/blob/main/packages/http-client-csharp/generator/docs/cadl-ranch.md#writing-cadlranchtests) which explains this a bit more.

And this [example PR](https://github.com/microsoft/typespec/pull/4241/files#diff-67d52e226c8af13c0c4136bf312fccad9e1838f640802d6c3d4434bd8cb29dac) as reference to the files that should be included.

**Comment by nisha-bhatia** [comment]
> hey @nisha-bhatia - looks like this pr is missing the actual test file to test the generated test project and also the update to the `launchSettings.json` file to ensure the cadl ranch test can be executed through the debugger. Here are some of the existing tests for reference: https://github.com/microsoft/typespec/tree/main/packages/http-client-csharp/generator/TestProjects/CadlRanch.Tests/Http/Client/Structure and the [doc](https://github.com/microsoft/typespec/blob/main/packages/http-client-csharp/generator/docs/cadl-ranch.md#writing-cadlranchtests) which explains this a bit more.
> 
> And this [example PR](https://github.com/microsoft/typespec/pull/4241/files#diff-67d52e226c8af13c0c4136bf312fccad9e1838f640802d6c3d4434bd8cb29dac) as reference to the files that should be included.

Thank you I totally missed this!

File: packages/website/.docusaurus/DONT-EDIT-THIS-FOLDER
Comment (timotheeguerin) [inline packages/website/.docusaurus/DONT-EDIT-THIS-FOLDER:1]: I think you didn';t meant to include this whole folder

**Comment by nisha-bhatia** [comment]
Closing this pr due to unwanted changes

**Pull request closed without merging**